### PR TITLE
Update: Latest Data Releases and Gene Ontology Changes

### PR DESCRIPTION
**Pull request by NegarJanani**
This pull request includes:
- Updates with the latest data releases.
- Reduced the number of annotations by 20 based on recent Gene Ontology updates.

Please review and let me know if any further adjustments are needed.

### CODE REVIEW

**Comment by dhimmel** [comment]
Nice. It looks good on a first pass but I will do a more thorough review. It's great to see some of the annotation counts grow.

Note that there is a conflict that must be resolved before I can merge. This conflict likely occurred since you began the PR on a branch that had a different commit than what got merged in this repository.

If you can create a new merge commit that merges in the dhimmel `gh-pages` branch, then I should be able to merge the PR on GitHub. You can probably just resolve conflicts be keeping the `code/process.ipynb` version that you currently have in this PR.

**Comment by NegarJanani** [comment]
I believe the merge conflicts have been successfully resolved. This branch should now be ready for review and merging. Please let me know if any further changes are needed.

File: summaries/summary-allev-inferred.json [hunk 1]
```diff
@@ -3,316 +3,141 @@
...
+      772427,
       "annotations/taxid_3702/GO_annotations-3702-inferred-allev.tsv"
     ],
-    [
-      4528,
-      "Oryza longistaminata",
-      30,
-      30,
-      "annotations/taxid_4528/GO_annotations-4528-inferred-allev.tsv"
-    ],
```
Comment (dhimmel) [inline summaries/summary-allev-inferred.json:16]: Any idea why several rice species like [this one](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=4528) no longer produce any output? Is it that they no longer have gene ontology annotations?

Perhaps the annotations all belong now to _Oryza glaberrima_?

Comment (NegarJanani) [reply summaries/summary-allev-inferred.json:16]: As I learned, most of the rice species are still listed in Gene Ontology, but they may no longer be present in Gene Entrez, which we are using. This could happen for several reasons:

Merging with similar species: Some species may have been merged with more widely studied species, such as Oryza glaberrima, especially if they share a significant amount of genomic information. In such cases, the annotations might be consolidated into the primary species, causing the others to no longer appear separately.

Lack of recent annotations: Taxa may be removed from Gene Entrez if they haven't had any recent updates or annotations. If there is a lack of new genomic data or functional annotations, these species might be flagged for removal or archiving.

Data reclassification: Sometimes, data undergoes reclassification or a shift in how species are categorized, and some taxons may no longer meet the criteria for inclusion in Gene Entrez, leading to their removal.

In summary, while the annotations may still exist in Gene Ontology, the species may no longer appear in Gene Entrez either due to merging, lack of recent updates, or reclassification.

Comment (NegarJanani) [reply summaries/summary-allev-inferred.json:16]: FYI, I'm working with a time constraint and would really appreciate it if you could expedite the review of the pull request.

The only change I made in the process.ipynb file is in Cell 8, where I defined the list of 45 TaxIds available in Websurface.

Thanks for your understanding and for helping me review this quickly!

**Review by dhimmel — APPROVED** [review]
Great work. It will be great to have these updates.

**Comment by dhimmel** [comment]
Everything looks good at https://git.dhimmel.com/gene-ontology/

**Comment by NegarJanani** [comment]
Perfect!
